azdev/operations/setup.py

```diff
@@ -83,27 +83,55 @@ def _install_cli(cli_path):
         "install -q -r {}/requirements.txt".format(cli_path),
         "Installing `requirements.txt`..."
     )
+    if deps == 'setup':
+        # command modules have dependency on azure-cli-core so install this first
+        pip_cmd(
+            "install -q -e {}/src/azure-cli-nspkg".format(cli_path),
+            "Installing `azure-cli-nspkg`..."
+        )
+        pip_cmd(
+            "install -q -e {}/src/azure-cli-telemetry".format(cli_path),
+            "Installing `azure-cli-telemetry`..."
+        )
+        pip_cmd(
+            "install -q -e {}/src/azure-cli-core".format(cli_path),
+            "Installing `azure-cli-core`..."
+        )
 
-    # command modules have dependency on azure-cli-core so install this first
-    pip_cmd(
-        "install -q -e {}/src/azure-cli-nspkg".format(cli_path),
-        "Installing `azure-cli-nspkg`..."
-    )
-    pip_cmd(
-        "install -q -e {}/src/azure-cli-telemetry".format(cli_path),
-        "Installing `azure-cli-telemetry`..."
-    )
-    pip_cmd(
-        "install -q -e {}/src/azure-cli-core".format(cli_path),
-        "Installing `azure-cli-core`..."
-    )
+        # azure cli has dependencies on the above packages so install this one last
+        pip_cmd("install -q -e {}/src/azure-cli".format(cli_path), "Installing `azure-cli`...")
+        pip_cmd(
+            "install -q -e {}/src/azure-cli-testsdk".format(cli_path),
+            "Installing `azure-cli-testsdk`..."
+        )
+    else:
+        # command modules have dependency on azure-cli-core so install this first
+        pip_cmd(
+            "install -e {}/src/azure-cli-nspkg --no-deps".format(cli_path),
+            "Installing `azure-cli-nspkg`..."
+        )
+        pip_cmd(
+            "install -e {}/src/azure-cli-telemetry --no-deps".format(cli_path),
+            "Installing `azure-cli-telemetry`..."
+        )
+        pip_cmd(
+            "install -e {}/src/azure-cli-core --no-deps".format(cli_path),
+            "Installing `azure-cli-core`..."
+        )
 
-    # azure cli has dependencies on the above packages so install this one last
-    pip_cmd("install -q -e {}/src/azure-cli".format(cli_path), "Installing `azure-cli`...")
-    pip_cmd(
-        "install -q -e {}/src/azure-cli-testsdk".format(cli_path),
-        "Installing `azure-cli-testsdk`..."
-    )
+        # azure cli has dependencies on the above packages so install this one last
+        pip_cmd("install -e {}/src/azure-cli --no-deps".format(cli_path), "Installing `azure-cli`...")
+        pip_cmd(
+            "install -e {}/src/azure-cli-testsdk --no-deps".format(cli_path),
+            "Installing `azure-cli-testsdk`..."
+        )
+        import platform
+        system = platform.system()
```

Comment:
Why not?
```suggestion
        system = platform.system().lower()
```

Reply:
For Linux/Mac, need capitalized first letter. I have verified that on Windows, `*.Windows.txt` also works when the file name is `*.windows.txt`. So I removed the lower() operation.